packages/hub/src/app/users/[username]/models/[slug]/ModelPage.tsx

```diff
@@ -89,15 +89,14 @@ export const ModelPage: FC<Props> = ({ username, slug, children }) => {
   const { data: session } = useSession();
 
   return (
-    <WithTopMenu>
-      <div className="flex items-center gap-4 max-w-2xl mx-auto">
+    <WithTopMenu addMarginToMainSection={false}>
+      <div
+        className="flex items-center gap-4 px-8 pt-5 pb-4 border-b border-gray-300"
+        style={{ backgroundColor: "#eceef0" }}
```

Comment:
I don't understand why you didn't like any of other `{gray}-100`, tbh. Because of the way tabs and `...` button looks on other backgrounds? If so, I hope we'll change it when we add new tab styles.

Reply:
This is basically in-between the {gray}-100 and the {gray}-200. It's right above the gray menu, which is awkward. 

I imagine later this will go through more revisions.